src/controls/qml/IntSelector.qml

```diff
@@ -32,40 +34,61 @@ Row {
     property string unitMarker: "%"
     // initial value of the control
     property int value: 0
-    // labelWidthRatio is the width of the label with respect to the total width
-    property real labelWidthRatio: 0.42857
-    // fontToHeightRatio is the size of the font relative to the height
-    property real fontToHeightRatio: 0.3
 
-    IconButton { 
-        iconName: "ios-remove-circle-outline"
+    Item {
+        id: positioner
+
+        width: parent.width
         height: parent.height
-        width: height
-        onClicked: {
-            var newVal = value - stepSize
-            if(newVal < min) newVal = min
-            value = newVal
+
+        IconButton {
+            id: buttonLeft
+
+            iconName: "ios-remove-circle-outline"
+            anchors {
+                left: parent.left
+                leftMargin: Dims.w(15)
+                verticalCenter: parent.verticalCenter
+            }
+            height: Dims.h(20)
```

Comment:
Rather than fixing all of the dimensions and making them immutable, it might be better to provide reasonable defaults, but then expose them as a `property alias`.  That was the intent behind the original ratio properties.

Reply:
Agreed, that is a good idea. Thanks for having helped also here.